### ISSUE TITLE
Version 1.6 maybe include some breaking changes

### ISSUE THREAD

**Issue by antonpirker** [pallets-eco/blinker#91]
Hey @pgjones!

I maintain the Sentry Python Repo and we run tests to make sure our SDK works with the Quart framework. Quart uses Blinker.

With the new release 1.6 our tests fail with the Blinker error `RuntimeError: Cannot send to a coroutine function`:
https://github.com/getsentry/sentry-python/actions/runs/4596847777/jobs/8137198226#step:5:608

This is the line in the test that fails:
https://github.com/getsentry/sentry-python/blob/master/tests/integrations/quart/test_quart.py#L147

I am not an expert in Blinker nor Quart, so I do not know why this is happening. 

I just wanted to bring to your attention that some behavior changed with the 1.6 release.

**Comment by davidism** [comment]
Please include a minimal reproducible example in the description when reporting an issue. Pointing to a line in the Sentry tests isn't easy for us to debug.

**Comment by davidism** [comment]
@antonpirker Can you follow up with more details as requested please?

**Comment by sl0thentr0py** [comment]
hey @davidism, here's a simple repro 
https://github.com/sl0thentr0py/sentry_sample_apps/tree/main/python/quart

run with 
```bash
python -m venv venv
source venv/bin/activate
pip install -r requirements.txt
python run.py
```

and go to `localhost:5000/health`

**Comment by davidism** [comment]
Please include code for a minimal reproducible example _directly_ in this issue, not as a link elsewhere.

**Comment by braian87b** [comment]
I'm getting an issue with newer version while using it on Flask, with Python 3.7:

```
  File "flask/__init__.py", line 5, in <module>
    from .app import Flask as Flask
  File "flask/app.py", line 34, in <module>
    from . import cli
  File "flask/cli.py", line 21, in <module>
    from .helpers import get_debug_flag
  File "flask/helpers.py", line 20, in <module>
    from .signals import message_flashed
  File "flask/signals.py", line 4, in <module>
    from blinker import Namespace
  File "blinker/__init__.py", line 1, in <module>
    from blinker.base import ANY
  File "blinker/base.py", line 18, in <module>
    from blinker._utilities import annotatable_weakref
  File "blinker/_utilities.py", line 1, in <module>
    import asyncio
  File "asyncio/__init__.py", line 21, in <module>
    from .base_events import *
  File "asyncio/base_events.py", line 296
    future = tasks.async(future, loop=self)
                       ^
SyntaxError: invalid syntax
```

**Comment by davidism** [comment]
That points to a line in the standard library, suggesting you have a problem with your install. Please report separately issues separately.

**Comment by braian87b** [comment]
I rebuild the image by just changing the version on the requirements to 1.5 and it worked... I build it again with 1.6 and it failed... Thanks

**Comment by pgjones** [comment]
Yep, there is an upgrade to Quart 0.19 for this - https://github.com/pallets/quart/commit/c9b555ff284bb54a45adab44c8e6d78ab8713da0, but I've also released 0.18.4 which restricts the blinker version.
I've also opened a PR for Sentry, https://github.com/getsentry/sentry-python/pull/1996, that should support the new Quart version and existing ones. Closing this issue, can track in sentry or Quart.
